index.js

```diff
@@ -9,6 +9,7 @@ const apmRouter = require('./lib/apm-router')
 const networks = JSON.parse(process.env.APMSERVE_NETWORKS)
 const aliases = JSON.parse(process.env.APMSERVE_ALIASES || '[]')
 
+app.use(require('helmet')())
```

Comment:
Didn't enable CSP because the apps can load from any arbitrary IPFS gateway and ETH node